src/main/java/org/terasology/metalrenegades/world/rivers/RiverToElevationProvider.java

```diff
@@ -62,20 +62,24 @@ public void process(GeneratingRegion region) {
         for (int i = 0; i < surfaceHeights.length; ++i) {
             float steepness = steepnessData[i];
             float riverFac = TeraMath.clamp(riversData[i]);
-            float riverBedElevation =
-                    seaLevel - rivers.maxDepth * (riversData[i] * (4 + 4 * steepness) - 3 - 4 * steepness);
+
```

Comment:
Could you please add the link to the graph visualization with a short comment here - I think it is really helpful to understand what is going on, and how some parameters affect the overall result.